tests/web/url.py

```diff
@@ -80,28 +80,50 @@ def test_param_get(self):
         not_here = u.qs_get('not_here')
         self.assertEqual(not_here, None)
 
-    def test_quote(self):
-        u = URL(quote('/path/to/+'))
-        self.assertEqual(u, '/path/to/%2B')
-        u = u.qs_set(page=7)
-        self.assertEqual(u, '/path/to/%2B?page=7')
+    #def test_quote(self):
+    #    # XXX this is WRONG!
+    #    # We shold not try to unquote the urlqouted values!
+    #    # Otherwise it is impossible to build a consistent interface
+    #    # If you want magic - use URL.from_url method!
+    #    u = URL(quote('/path/to/+'))
+    #    self.assertEqual(u, '/path/to/%2B')
+    #    u = u.qs_set(page=7)
+    #    self.assertEqual(u, '/path/to/%2B?page=7')
```

Comment:
I feel uncomfortable here. Should the interface be encoded (urlquotted) or unicode? I see only unicode interface of all parts, because urlencoded interface breaks working with query strings (`qs_set`).

Any suggestions?